### PR TITLE
Taking chatgpt for an impromptu spin

### PR DESCRIPTION
**Pull request by matthewreed26**

### CODE REVIEW

File: posts/taking-chatgpt-for-an-impromptuspin.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+  - ChatGPT
+  - AI
+date: 2023-01-23T12:00:00.000Z
+title: 'Taking ChatGPT for an Impromptu Spin'
+image:
+---
+
+## Introduction
+
+[OpenAI](https://openai.com/blog/chatgpt/)'s brainchild, "[ChatGPT](https://chat.openai.com/)," is all the rage right now and is identified/poised to be a massive disrupter in the software engineering industry. Since its release on November 30th, people have been writing plenty of guides, blogs, and opinion pieces about it all over social media. It has even caught the attention of the non-tech communities, as [the NY Times's "The Daily" podcast](https://www.stitcher.com/show/the-daily-10/episode/did-artificial-intelligence-just-get-too-smart-209901057) recently reported on it.
```
Comment (pkrish123) [inline posts/taking-chatgpt-for-an-impromptuspin.md:14]: "identified/poised" reads oddly to me here. I would pick one or the other.

File: posts/taking-chatgpt-for-an-impromptuspin.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+
+## Conclusion
+
+As [Microsoft continues to funnel money into the ChatGPT project](https://edition.cnn.com/2023/01/23/tech/microsoft-invests-chatgpt-openai/index.html) and [Google has its version of "Deepmind Chat" coming soon](https://www.techradar.com/news/googles-deepmind-promises-chatgpt-rival-soon-and-it-could-be-better-in-one-key-way), it is clear the days of artificial intelligence are only just beginning. Many industries have yet to see automation like this before and will face a [partial or complete overhaul](https://80000hours.org/2015/02/which-careers-will-be-automated/) soon.
+
+```console
+Hello Brave New World?
```
Comment (pkrish123) [inline posts/taking-chatgpt-for-an-impromptuspin.md:93]: I love this response! It really ties in with how alien, and foreign ChatGPT's self-generated origin story is!

**Review by pkrish123 — CHANGES_REQUESTED** [review]
I love this! Maybe you can add a small blurb before the conclusion pointing people to try ChatGPT themselves, and how to install it?

**Comment by matthewreed26** [comment]
> I love this! Maybe you can add a small blurb before the conclusion pointing people to try ChatGPT themselves, and how to install it?

Sure, I can encourage readers to go signup for a free account and try it out for themselves! But no installing as it is web-hosted haha.

**Comment by pkrish123** [comment]
> > I love this! Maybe you can add a small blurb before the conclusion pointing people to try ChatGPT themselves, and how to install it?
> 
> Sure, I can encourage readers to go signup for a free account and try it out for themselves! But no installing as it is web-hosted haha.

Oh! Then the console parts are just the web UI! got it!

**Review by pkrish123 — APPROVED** [review]

**Pull request merged**